internal/storage/v1/elasticsearch/options.go

```diff
@@ -171,6 +176,10 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 		tokenPath,
 		"Path to a file containing bearer token. This flag also loads CA if it is specified.")
 
+	flagSet.Duration(
+		nsConfig.namespace+suffixBearerTokenReloadInterval,
+		reloadInterval,
+		"Interval for reloading bearer token from file. Set to 0 to disable automatic reloading.")
```

Comment:
according to tokenloader 0 means "reload on every request", not "disable reloading". I think this definition ^ is correct but the implementation is not.